Added note about unsuported expressions in Python parameters file (#2772)
iterative/dvc.org: content/docs/command-reference/params/index.md

## Examples: Python parameters file

> > ⚠️ Note that complex expressions (those not supported by
> > [ast.literal_eval](https://docs.python.org/3/library/ast.html#ast.literal_eval))
> > inside the Python parameters file won't be parsed.

Reviewer: ```suggestion
> ⚠️ Note that complex expressions (unsupported by
> [ast.literal_eval](https://docs.python.org/3/library/ast.html#ast.literal_eval))
> won't be parsed as DVC parameters.
```

TUPLE = (10, 100)
NONE = None

> DICT_EXP = dict(a=1, b=2) # Will NOT be found due to the expression

Reviewer: ```suggestion
# Complex expressions will be ignored.
DICT_EXP = dict(a=1, b=2)
```

TUPLE = (10, 100)
NONE = None

# Complex expressions will be ignored.

> DICT_EXP = dict(a=1, b=2)

Reviewer: Is that correct? Will they be ignored or will there be some sort of error @daavoo ? Thanks

Author: It is currently ignored. I'm thinking about sending a P.R. to at least log a warning
